src/components/talk/speaker_name.js

```diff
@@ -0,0 +1,19 @@
+import React from "react"
+
+import SpeakerNameStyles from "../../styles/talk/speaker_name.module.css"
+
+const SpeakerName = ({ name, occupations }) => {
+  return (
+    <div className={SpeakerNameStyles.container}>
+      <div className={SpeakerNameStyles.name_container}>
+        <div className={SpeakerNameStyles.name}>{name}</div>
+        <hr />
+      </div>
+      <div className={SpeakerNameStyles.occupation}>
+        {"@" + occupations[0].where}
+      </div>
```

Comment:
Shouldn't we show every occupation here. There's a occupations component that already does that. You just need to include it here.